data/config/delft.json

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"NAME": "default", "city": "Delft, Netherlands", "nP": 20, "nV": 5, "times": {"request": 15, "transaction": 20, "pickup": 30, "dropoff": 10, "patience": 600, "pickup_patience": 90}, "speeds": {"walk": 1.2, "ride": 10}, "t0": "17:00", "paths": {"main": "/Users/rkucharski/Documents/GitHub/MaaSSim", "data": "/Users/rkucharski/Documents/GitHub/MaaSSim/data", "params": "/Users/rkucharski/Documents/GitHub/MaaSSim/data/configs", "G": "../../data/graphs/Delft.graphml", "skim": "../../data/graphs/Delft.csv", "_ipython_display_": {}, "_repr_mimebundle_": {}, "_repr_html_": {}, "_repr_markdown_": {}, "_repr_svg_": {}, "_repr_png_": {}, "_repr_pdf_": {}, "_repr_jpeg_": {}, "_repr_latex_": {}, "_repr_json_": {}, "_repr_javascript_": {}}, "dist_threshold": 100000, "big_time": 10, "sleep": false, "simTime": 1, "demand_structure": {"origins_dispertion": -0.0003, "destinations_dispertion": -0.001, "temporal_dispertion": 0.3, "temporal_distribution": "uniform"}, "shareability": {"windows": {"pickup": [-2, 5], "dropoff": [-10, 5]}, "avg_speed": 10, "shared_discount": 0.8, "VoT": 0.005, "WtS": 1.2, "delay_value": 1.5, "price": 1.5, "matching_obj": "u_veh", "pax_delay": 45, "horizon": 600, "max_degree": 600, "share": 0}, "parallel": {"nThread": 3, "nReplications": 2}, "nD": 1, "_ipython_display_": {}, "_repr_mimebundle_": {}, "_repr_html_": {}, "_repr_markdown_": {}, "_repr_svg_": {}, "_repr_png_": {}, "_repr_pdf_": {}, "_repr_jpeg_": {}, "_repr_latex_": {}, "_repr_json_": {}, "_repr_javascript_": {}, "platforms": {"base_fare": 1.4, "km_fare": 1.21, "min_fare": 0, "comm_rate": 0.25}}
```

Comment:
What was changed here?

Reply:
I added `params.platforms` fare parameters. But I will remove if you don't want it to be in the main MaaSSim simulator.